lua/telescope/builtin/files.lua

```diff
@@ -249,11 +248,10 @@ files.file_browser = function(opts)
     return finders.new_table {
       results = data,
       entry_maker = (function()
-        local tele_path = require'telescope.path'
         local gen = make_entry.gen_from_file(opts)
         return function(entry)
           local tmp = gen(entry)
-          tmp.ordinal = tele_path.make_relative(entry, opts.cwd)
+          tmp.ordinal = Path:new(entry):make_relative(opts.cwd)
```

Comment:
Hmm `Telescope file_browser` is broken because dirs no longer have the os sep at the end of the file. We need to make sure that it stays there.

I dont even this this change is coursing it but changes in `gen_from_file`. Maybe we should write a completly new `entry_maker` for file_browser. What do you think?

Reply:
Yeah it's not 100% clear to me what you mean, but I'd be happy to take a crack at it later today :)

Reply:
Sorry i just did it myself :)